app/database/deprovision/template.hbs

```diff
@@ -0,0 +1,35 @@
+<div class="row">
+  <div class="col-xs-9">
+    <div class="panel panel-default panel-subdued panel-danger">
+      <div class="panel-heading">
+        <h3>Permanently deprovision {{model.handle}} </h3>
+      </div>
+      <div class="panel-body">
+        <form role="form">
+          {{#if error}}
+            <div class="alert alert-warning fade in">
+              <a class="close" data-dismiss="alert">×</a>
+              <p>
+                There was a problem deprovisioning your database: {{error.responseJSON.message}}
```

Comment:
We should eventually settle on some sort of general strategy for displaying errors. Errors that don't have this shape (like a syntax error or some generic 500 response from the server) would end up looking funny.